package.json

```diff
@@ -95,7 +95,7 @@
     "ts-node": "^10.4.0",
     "ts-toolbelt": "^9.6.0",
     "typedoc": "^0.20.36",
-    "typescript": "~4.1.5"
+    "typescript": "4.2"
```

Comment:
We should keep the same version as in the other packages: `~4.1.5`

Reply:
fixed.